### DISCUSSION TITLE
Use Python Launcher on Unix

### DISCUSSION THREAD

**Discussion by peterchristofferholm** [quarto-dev/quarto-cli#5917] [Feature Requests]
### Description

As per the documentation, Quarto will select automatically select a version of Python using the [Python Launcher](https://docs.python.org/3/using/windows.html#python-launcher-for-windows) on Windows, but on MacOS and Linux it uses the system PATH. There exists a 3rd party port of the Python Launcher for Unix (https://github.com/brettcannon/python-launcher) which works great. It would be great if Quarto could support this on MacOS and Linux.

**Comment by mcanouil** [comment]
Thanks for the suggestion.
We'll update here or on a specific issue regarding this in the future.
